pkgs/top-level/all-packages.nix

```diff
@@ -15430,6 +15432,12 @@ in
 
   openmw = callPackage ../games/openmw { };
 
+  inherit (callPackages_i686 ../games/openrct2 {
+      stdenv = stdenv_32bit;
+    })
+    openrct2
+    openrct2-engine;
+
```

Comment:
if you add both to top-level, then why bother creating a set first in `default.nix` and then inheriting from that set?